server/routes/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const mainController = require("../controllers/mainController");
+
+<<<<<<< HEAD
+// router.get("/", mainController.homePage);
+
+router.get('/', (req, res) => {
+  res.render('index');
+});
+=======
+router.get("/", mainController.novelsPage);
+>>>>>>> b00a69c... add templating
```

Comment:
Looks like there's some left over symbols from a merge conflict which need to be removed. Also can line 6 be removed now?

Reply:
cleaned